snuba/datasets/storages/factory.py

```diff
@@ -168,10 +151,18 @@ def get(self, storage_key: StorageKey) -> Storage:
         return self._all_storages[storage_key]
 
     def get_writable_storage_keys(self) -> list[StorageKey]:
-        return list(self._writable_storages.keys())
+        return [
```

Comment:
do you have a test to see if you're fixing the bug this PR was supposed to fix?

```
This PR fixes a consumer [CLI](https://github.com/getsentry/snuba/blob/ff935eeb47b56fe54b0fda294b9055ab0acfcf48/snuba/cli/consumer.py#L44-L52) bug which was not returning the writable storages defined in config.
```

Reply:
Added file and a test for ensuring config-built writable storage key exists in get_writable_storage_keys()